test/videos/patch.test.js

```diff
@@ -0,0 +1,492 @@
+/* eslint-disable max-lines */
+const { getResponse, truncateUserCollection, truncateVideoCollection } = require('../utils/utils');
+const { mockValidateTokenAndLoadUser } = require('../mocks/authorization');
+// const { mockUploadVideo, mockFailUploadVideo } = require('../mocks/videos');
+const { userDataFactory, createUserFactory, friendFactory } = require('../factories/users');
+const { TOKEN_FOR_AUTH } = require('../utils/constants');
+const { createVideoFactory } = require('../factories/videos');
+const Videos = require('../../app/models/video');
+
+const likeEndpoint = id => `/videos/${id}/like`;
+const dislikeEndpoint = id => `/videos/${id}/dislike`;
+const unlikeEndpoint = id => `/videos/${id}/unlike`;
+const undislikeEndpoint = id => `/videos/${id}/undislike`;
+
+const videoData = {
+  title: 'AVideoTitle',
+  description: 'AVideoDescription',
+  download_url: 'https://someUrl.com',
+  datetime: '2020-05-18T18:43:35',
+  visibility: 'private',
+  file_name: 'video.mp4',
+  file_size: '24335',
+  latitude: 50,
+  longitude: 50,
+  videoId: 1
+};
+
+describe('PATCH /videos/:id/like', () => {
+  describe('Like unexistent video', () => {
+    let unexistentRes = {};
+    const unexistentId = 2000;
+    const userData = userDataFactory();
+
+    beforeAll(async () => {
+      await truncateUserCollection();
+      await truncateVideoCollection();
+      await createUserFactory(userData.username);
+      mockValidateTokenAndLoadUser(userData);
+      unexistentRes = await getResponse({
+        endpoint: likeEndpoint(unexistentId),
+        method: 'patch',
+        header: { authorization: TOKEN_FOR_AUTH }
+      });
+    });
+
+    afterAll(() => jest.clearAllMocks());
+
+    it('Check status', () => {
+      expect(unexistentRes.status).toBe(409);
+    });
+
+    it('Check internal code', () => {
+      expect(unexistentRes.body.internal_code).toBe('video_not_exists');
+    });
+
+    it('Check message', () => {
+      expect(unexistentRes.body.message).toBe(`Video with id ${unexistentId} does not exist`);
+    });
+  });
+
+  describe('like private video of a friend', () => {
```

Comment:
Confuso el nombre del test. Creo que quisite decir "Like private video of another non-friend user"